kolibri/core/assets/src/core-global.styl

```diff
@@ -106,3 +106,8 @@ button::-moz-focus-inner
   padding: 0
   position: absolute
   width: 1px
+
+.description
+  margin-top: 2em
+  margin-bottom: 1.4em
+  line-height: 1.8em
```

Comment:
Hey - could you move this class to:

    kolibri/plugins/learn/assets/src/vue/learn.styl

thanks!